templates/functions/webhook/handler.ts.ejs

```diff
@@ -0,0 +1,34 @@
+/**
+ * This file was generated using 8base CLI.
+ * 
+ * To learn more about writing custom GraphQL resolver functions, visit
+ * the 8base documentation at:
+ * 
+ * https://docs.8base.com/8base-console/custom-functions/webhooks
+ *
+ * To update this functions invocation settings, update its configuration block
+ * in the projects 8base.yml file:
+ *  functions:
+ *    <%= functionName %>:
+ *      ...
+ *
+ * Data that is sent to the function can be accessed on the event argument at:
+ *  event.data[KEY_NAME]
+ *
+ * To invoke this function locally, run:
+ *  8base invoke-local <%= functionName %> -p src/webhooks/<%= functionName %>/mocks/request.json
+ */
+
+type WebhookResult = {
+  data: {
+    result: string,
+  },
+};
+
+export default async (event: any, ctx: any) : Promise<WebhookResult> => {
+  return {
+    data: {
```

Comment:
@zouxuoz Are you sure about return value format? I believe it should be like `{ statusCode: 200, body: JSON.stringify({}) }`.

Reply:
@vorobeez fixed